tests/integration/ledger/mocks/acc_retriever.go

```diff
@@ -0,0 +1,29 @@
+package mocks
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ client.AccountRetriever = (*MockAccountRetriever)(nil)
+
+// MockAccountRetriever defines a no-op basic AccountRetriever that can be used
+// in mocked contexts. Tests or context that need more sophisticated testing
+// state should implement their own mock AccountRetriever.
+type MockAccountRetriever struct{}
```

Comment:
is this supposed to return nil values for everything? It's better if you return a non-nil account with the address